### PR TITLE
log failures to read certificates during listing

### PR DESCRIPTION
**Pull request by icewind1991**
And check that we could actually read the file

### CODE REVIEW

**Review by kesselb — APPROVED** [review]

File: lib/private/Security/CertificateManager.php [hunk 1]
```diff
@@ -83,8 +83,14 @@ public function listCertificates(): array {
 		while (false !== ($file = readdir($handle))) {
 			if ($file != '.' && $file != '..') {
 				try {
-					$result[] = new Certificate($this->view->file_get_contents($path . $file), $file);
+					$content = $this->view->file_get_contents($path . $file);
+					if ($content) {
+						$result[] = new Certificate($content, $file);
+					} else {
+						$this->logger->error("Failed to read certificate from $path");
```
Comment (Fenn-CS) [inline lib/private/Security/CertificateManager.php:90]: Is this really necessary? file contents can return a non-boolean value that evaluates to false, would this statement still be correct?

**Review by nfebe — APPROVED** [review]
I left an inline comment/question, otherwise it looks good. Thanks!

File: lib/private/Security/CertificateManager.php (hunk 1 above)
Comment (icewind1991) [reply lib/private/Security/CertificateManager.php:90]: Switched to a strict check

**Pull request merged**